dapp/src/pages/requests.tsx

```diff
@@ -1,15 +1,41 @@
-import React from "react"
-import { Link } from "gatsby"
+import React, { useEffect } from "react"
+import { graphql, PageProps, useStaticQuery } from "gatsby"
 
 import SEO from "../components/seo"
+import Tabs from "../components/tabs"
 
-const RequestList = () => (
-  <>
-    <SEO title="Requests" />
-    <h2>Requests</h2>
-    <p>()</p>
-    <Link to="/">Go back to the homepage</Link>
-  </>
-)
+const REQUESTS_PAGE_QUERY = graphql`
+  query RequestsPage {
+    grantsdao {
+      systemInfo: systemInfos {
+        proposalCount
+        totalBalance
+      }
+    }
+  }
+`
 
-export default RequestList
+const RequestsPage: React.FC<PageProps> = () => {
+  const {
+    grantsdao: { systemInfo },
+  } = useStaticQuery(REQUESTS_PAGE_QUERY)
+
+  // TODO: Remove after implement it, this effect just exists for logging purpose.
```

Comment:
:point_right:

Reply:
I'll clean this up once I implement the requests page